invenio_oauthclient/contrib/cern_openid.py

```diff
@@ -104,6 +104,10 @@
     icon="",
     logout_url="https://auth.cern.ch/auth/realms/cern/protocol/"
     "openid-connect/logout",
+    signup_options={
```

Comment:
Default for `orcid` app

Reply:
minor: isn't this the default for remote apps set below?

Reply:
This CERN OpenID is a bit "special", fully custom :( It does not inherit from the OAuth helper as all the others.